main.cpp

```diff
@@ -194,19 +241,24 @@ struct VaccumCleaner
     std::string informUserWhenFull();
 };
 
-/*
-Thing 3) Bicycle
-5 properties:
-    1) number of gears (int)
-    2) type (std::string)
-    3) speed (int)
-    4) cadence (std::string)
-    5) color (std::string)
-3 things it can do:
-    1) cycle at a certain speed
-    2) turn left or right
-    3) break
- */
+void VaccumCleaner::dustCollect(int dustQuantity, int time)
+{
+    if (dustQuantity != 0) {
+        dustQuantity -= 1;
+        time -= 1;
+    }
+}
+
+bool VaccumCleaner::retractCord(int cordDistance)
+{
+    if (cordLength == 0)
+        return false;
+    else
+    {
+        cordLength -= cordDistance;
+        return true;
+    }
+}
```

Comment:
2 things to consider here:
1) If you use curly braces in an `if/else` block, use them in both the `if` and the `else` blocks.  It is easier to read
2) If you find yourself writing code like this:
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    else 
    {
        return anotherValue;
    }
}
```
You do not need the `else` block.  Here's why:
`return` statements are the last thing that **ever happens** in a function.
When the `if()` expression is evaluated as `true` and the block is entered, the `return aValue;` is the last line of code to be executed before the function exits.
but when the `if()` expression evaluates as `false` and the block is **NOT** entered, the next thing that happens is `return anotherValue;`
you have no code after the `else` block that needs executing, since your `else` block contains a `return` statement.
that means you do **NOT** need the `else` block, and it is just causing visual clutter.
```
int Type::someFunction()
{
    if( someExpression )
    {
        someStatement();
        return aValue;
    } 
    
    return anotherValue;
}
```
- [ ] revise